### DISCUSSION TITLE
Support maxFiles on Dropzone

### DISCUSSION THREAD

**Discussion by rodolphoasb** [mantinedev/mantine#1469] [Feature requests]
Currently, our Dropzone component doesn't support the `maxFiles` prop that is supported by the [react-dropzone](https://react-dropzone.js.org/) package.

Not sure but maybe this is just a type thing that we would need to add.

**Comment by rtivital** [comment]
I'll add this feature in next minor release

**Reply by rodolphoasb** [reply]
Hey, @rtivital! Sorry to bother... Any update on this one?

![image](https://user-images.githubusercontent.com/8127858/170994829-3b5e0d26-2b02-4232-83ce-a475ffaa4984.png)

**Reply by rtivital** [reply]
It will be added in next release (minor or major whichever comes out next).
